core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched/model/staff/Staff.kt

```diff
@@ -4,20 +4,20 @@ import kotlinx.collections.immutable.PersistentList
 import kotlinx.collections.immutable.toPersistentList
 
 data class Staff(
-    val id: String,
-    val name: String,
-    val icon: String,
-    val profileUrl: String?,
+    val id: Long,
+    val username: String,
+    val profileUrl: String,
+    val iconUrl: String,
 ) {
```

Comment:
This model class is shared with the iOS app. Could you also update KMPConverters.swift? Otherwise the iOS app might fail to build.

Reply:
@kitakkun CC @ry-itto 
I'll make the correction! 🫡 
One thing I'm concerned about is that when modifying the model, it seems like the iOS side's CI isn't running. Is this intentional?
#335

Reply:
When changes are made to Staff.swift, iOS CI runs, but when changes are made to Staff.kt (the source of Shared), iOS CI does not appear to run.
https://github.com/DroidKaigi/conference-app-2025/pull/388/commits/332d820592dab1bcf403dd794d6156b77e061584